app/models/payout_report.rb

```diff
@@ -4,7 +4,6 @@ class PayoutReport < ApplicationRecord
 
   LEGACY_PAYOUT_REPORT_TRANSITION_DATE = "2018-12-01 09:14:58 -0800".freeze
 
-  MINIMUM_BALANCE_AMOUNT = 5
```

Comment:
We should keep this and move this to about 0.01 USD, as users with 0 balance shouldn't see the message.

Reply:
JA no longer needed since we should not have any minimums for BF.